Examples/ObjectiveC/FillPatternExample.m

```diff
@@ -0,0 +1,69 @@
+#import "FillPatternExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleFillPattern = @"FillPatternExample";
+
+@interface FillPatternExample ()<MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation FillPatternExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Set the map’s size, style, center coordinate, and zoom level.
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.frame styleURL:[MGLStyle darkStyleURLWithVersion:9]];
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.849534447, -77.039222717)
+                       zoomLevel:8.5
+                        animated:NO];
+    mapView.delegate = self;
+    
+    [self.view addSubview:mapView];
+    
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Set the UIImage to be used for the fill pattern.
+    UIImage *fillPatternImage = [UIImage imageNamed:@"stripe-pattern"];
+    
+    // Add the fill pattern image to used by the style layer.
+    [style setImage:fillPatternImage forName:@"stripe-pattern"];
+    
+    // "mapbox://examples.0cd7imtl" is a map ID referencing a tileset. For more
+    // more information, see mapbox.com/help/define-map-id/
+    MGLSource *source = [[MGLVectorSource alloc] initWithIdentifier:@"drone-restrictions" configurationURL:[NSURL URLWithString:@"mapbox://examples.0cd7imtl"]];
+    
+    [style addSource:source];
+    
+    // Create a style layer to be used with the vector source.
```

Comment:
> to be used

Would “using” be more precise? We’ve already init’d the source on L38 and are adding it to the layer here.